addons/board/static/src/js/dashboard.js

```diff
@@ -347,6 +347,10 @@ core.form_tag_registry
 FavoriteMenu.include({
     start: function () {
         var self = this;
+        if(self.action_id === undefined) {
```

Comment:
You can use 'this' instead of 'self' here (we always use 'this' when possible to prevent unnecessary 'self' leftovers when code evolves.)